magicctapipe/scripts/lst1_magic/lst_m1_m2_cloud_correction.py

```diff
@@ -0,0 +1,417 @@
+                image /= trans_pixels
+                corr_image = image.copy()
+                corr_image[~cleanmask] = 0
+
+                hillas_params = hillas_parameters(camgeom[telid], corr_image)
+                timing_params = timing_parameters(
+                    camgeom[telid], corr_image, peak_time, hillas_params
+                )
+                leakage_params = leakage_parameters(
+                    camgeom[telid], corr_image, signal_pixels
+                )
+                conc_params = concentration_parameters(
+                    camgeom[telid], corr_image, hillas_params
+                )
+
+                source_pos_in_camera = sky_to_camera(
+                    alt_rad * u.rad,
+                    az_rad * u.rad,
+                    focal_eq[telid],
+                    pointing_alt * u.rad,
+                    pointing_az * u.rad,
+                )
+
+                disp_parameters = disp.disp(
+                    x, y, source_pos_in_camera.x, source_pos_in_camera.y, psi
+                )
+
+                disp_dx = disp_parameters[0].value * u.m
+                disp_dy = disp_parameters[1].value * u.m
+                disp_norm = disp_parameters[telid].value * u.m
+                disp_angle = disp_parameters[3].value * u.rad
+                disp_sign = disp_parameters[4]
+                src_x = source_pos_in_camera.x.value * u.m
+                src_y = source_pos_in_camera.y.value * u.m
+
+                data.append(
+                    {
+                        "tel_id": telid,
+                        "event_id": event_id,
+                        "obs_id": obs_id,
+                        "event_id_lst": event_id_lst,
+                        "obs_id_lst": obs_id_lst,
+                        "intensity": hillas_params.intensity,
+                        "x": hillas_params.x,
+                        "y": hillas_params.y,
+                        "r": hillas_params.r,
+                        "phi": hillas_params.phi,
+                        "length": hillas_params.length,
+                        "width": hillas_params.width,
+                        "psi": hillas_params.psi,
+                        "skewness": hillas_params.skewness,
+                        "kurtosis": hillas_params.kurtosis,
+                        "slope": timing_params.slope,
+                        "intercept": timing_params.intercept,
+                        "deviation": timing_params.deviation,
+                        "intensity_width_1": leakage_params.intensity_width_1,
+                        "intensity_width_2": leakage_params.intensity_width_2,
+                        "pixels_width_1": leakage_params.pixels_width_1,
+                        "pixels_width_2": leakage_params.pixels_width_2,
+                        "conc_cog": conc_params.cog,
+                        "conc_core": conc_params.core,
+                        "conc_pixel": conc_params.pixel,
+                        "impact": impact,
```

Comment:
you should not save parameters like impact, alt, az, etc., because they were not recomputed after the correction. 
The file that you save should have DL1-like format and then we need to run DL1stereo macro over it again to get corrected stereoscopic reconstruction

Reply:
Removed from the saving part.